pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -101,12 +103,15 @@ type AzDriverNode struct {
 
 // AzDriverNodeSpec is the spec for a AzDriverNode resource
 type AzDriverNodeSpec struct {
-	UnderlyingNode string `json:"underlyingNode"`
+	CSINodeID string `json:"underlyingCSINode"`
+	NodeName  string `json:"underlyingNode"`
```

Comment:
I thought node ID and name both uniquely identify the node with name more commonly used to reference nodes. Do we need both?

Reply:
For the purposes of this PR, CSINodeID is not required. I've added the field based on the CRD descriptions - @abhisheksinghbaghel could more definitely tell us if this field is used by any of the other components. We can remove the field in this PR and add it back when it is actually used.